pkg/apis/events/v1alpha1/cloudbuildsource_conversion.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/knative-gcp/pkg/apis/convert"
+	"github.com/google/knative-gcp/pkg/apis/events/v1beta1"
+	"knative.dev/pkg/apis"
+)
+
+// ConvertTo implements apis.Convertible.
+// Converts source (from v1beta1.CloudBuildSource) into v1alpha1.CloudBuildSource.
+func (source *CloudBuildSource) ConvertTo(_ context.Context, to apis.Convertible) error {
+	switch sink := to.(type) {
+	case *v1beta1.CloudBuildSource:
+		sink.ObjectMeta = source.ObjectMeta
+		sink.Spec.PubSubSpec = convert.ToV1beta1PubSubSpec(source.Spec.PubSubSpec)
+		sink.Spec.Topic = source.Spec.Topic
```

Comment:
v1beta1 should not have source.Spec.Topic... We should remove the topic from there, as it is always "cloud-build" (or something like that)... Users shouldn't be able to configure it. 
I created an issue for that. 
https://github.com/google/knative-gcp/issues/1211

Reply:
Currently for CloudBuildSource, the logic is as follows:
https://github.com/google/knative-gcp/blob/7a9e760e7dd294a0e0e9fb5f89208f028b5cd9f8/pkg/apis/events/v1alpha1/cloudbuildsource_types.go#L57
We just put it in spec for user's reference.

Reply:
// It is optional. Defaults to 'cloud-builds' and the topic must be 'cloud-builds'

Reply:
We can delete that if it is not needed